src/App.tsx

```diff
@@ -33,14 +34,14 @@ export default class App extends React.Component<unknown, AppState> {
     const thirdPartyAuthDetails = await sessionsClient.createUserSession({
       successURL: window.location.href
     })
+    await getUser()
```

Comment:
I'm not clear on the reason for this call here. It will have no effect.

Reply:
This function not only fetches the userId but the main use of this function is to update the localStorage accordingly. and same is the case with the removeUser function. 

Expected behaviour is to fetcch the userId on login and store it in localStorage. That's what I was trying to do.

Reply:
You do not have a valid token at the point in time where you are invoking this.

Reply:
Ohh ! I remember you told me that page will refresh in between. So, should I put this invocation in the constructor ?

Reply:
I would just remove this call. The way the `getUser()` function was written, it will lazily fetch the info the first time it is called.

Reply:
Ohhk ! Understood. I will just remove the invocation of getUser function. 👍

Reply:
> I remember you told me that page will refresh in between.

Yes... we redirect to the third-party identity provider and when the user is properly authenticated over there, they call back to the API server to activate the token. And that redirects back to this page.

So after the authn process is completed, the user comes _back_ in the door of the dashboard with a newly-activated token in local storage.

Reply:
Yaa ! I forgot that part. Sorry My bad.